ProjectLighthouse.Servers.Website/Pages/Debug/FilterTestPage.cshtml.cs

```diff
@@ -19,7 +19,7 @@ public FilterTestPage(DatabaseContext database) : base(database)
     public IActionResult OnGet(string? text = null)
     {
         #if DEBUG
-        if (text != null) this.FilteredText = CensorHelper.FilterMessage(text);
+        if (text != null) this.FilteredText = CensorHelper.FilterMessage(text, "test filter", null);
```

Comment:
```suggestion
        if (text != null) this.FilteredText = CensorHelper.FilterMessage(text, "test filter");
```
The default value is already `null`

Reply:
Admittedly I considered changing this but I just lazily didn't because there are already like 3 warnings in the tests that haven't been fixed for ages :clueless:.